worlds/banjo_tooie/Options.py

```diff
@@ -0,0 +1,513 @@
+    """Open world entrances without having to go to Jiggywiggy."""
+    display_name = "Skip Puzzles"
+
+class Backdoors(Toggle):
+    """Opens many one way switches on game start, allowing for more backdoor access to levels.
+    The following gates are preopened: MT>TDL, MT>HFP, GGM>WW, WW>TDL
+    For MT>TDL only the gate accessed from TDL's side is opened and for GGM>WW the boulders are still unexploded
+    The bridge from HFPs entrance is pre-moved to allow secondary access to clifftop
+    George is also pre-dropped to make HFP>JRL more accessible"""
+    display_name = "Open Backdoors"
+
+class OpenHag1(DefaultOnToggle):
+    """HAG 1 boss fight is opened when Cauldron Keep is opened. Only 55 jiggies are needed to win."""
+    display_name = "HAG 1 Open"
+
+class RandomizeWorlds(Toggle):
+    """Worlds will open in a randomized order. Randomized Moves and Puzzle Skip Required."""
+    display_name = "Randomize Worlds"
+
+class RandomizeWorldZones(Toggle):
+    """World Entrances will warp you to a different world. This does not affect Chuffy."""
+    display_name = "Randomize World Entrances"
+
+class RandomizeStopnSwap(Toggle):
+    """Mystery Eggs, their rewards, and the Ice Key are scattered across the MultiWorld."""
+    display_name = "Randomize Stop n Swap"
+
+class SkipToT(Choice):
+    """Choose whether to play the full quiz, start at round 3, or skip it."""
+    display_name = "Tower of Tragedy Quiz"
+    option_full = 0
+    option_skip = 1
+    option_round_3 = 2
+    default = 1
+
+class LogicType(Choice):
+    """Choose your logic difficulty and difficulty of tricks you are expected to perform to reach certain areas.
+    Please be aware that if you plan on randomizing worlds with BK Moves in the pool, you cannot use "intended" logic.""" 
+    display_name = "Logic Type"
+    option_intended = 0
+    option_easy_tricks = 1
+    option_hard_tricks = 2
+    option_glitches = 3
+    default = 0
+
+class Silos(Choice):
+    """Choose if you want IoH Silos to be closed, randomly open 1 or enable all. If you enabled Randomized Worlds with BK Moves randomized and
+       silos set to none, it will be enforced to one."""
+    display_name = "Open Silos"
+    option_none = 0
+    option_one = 1
+    option_all = 2
+    default = 0
+
+class VictoryCondition(Choice):
+    """Choose which victory condition you want
+    HAG1: Unlock the HAG1 fight and defeat Gruntilda
+    Minigame Hunt: Clear the 14 minigames and the final Canary Mary race in Cloud Cuckcoo Land to collect Mumbo Tokens
+    Boss Hunt: Kill the 8 world bosses and collect their Mumbo Tokens
+    Jinjo Family Rescue: Rescue Jinjo Families to collect their prized Mumbo Tokens
+    Wonderwing Challenge: Collect all 32 Mumbo Tokens across all boss fights, mini games and every Jinjo family
+        to gain access to HAG1 and Defeat Grunty. The Ultimate Banjo Tooie experience!!
+    Token Hunt: Mumbo's Tokens are scattered around the world. Help him find them"""
```

Comment:
```suggestion
    """Choose the victory condition.
    HAG1: Unlock the HAG1 fight and defeat Gruntilda.
    Minigame Hunt: Clear the 14 minigames and the final Canary Mary race in Cloud Cuckcoo Land to collect Mumbo Tokens.
    Boss Hunt: Defeat the 8 world bosses and collect their Mumbo Tokens.
    Jinjo Family Rescue: Rescue Jinjo Families to collect their prized Mumbo Tokens.
    Wonderwing Challenge: Collect all 32 Mumbo Tokens across all boss fights, mini games, and every Jinjo family
        to gain access to HAG1 and Defeat Grunty. The Ultimate Banjo Tooie experience!!
    Token Hunt: Mumbo's Tokens are scattered around the world. Help him find them!"""
```
Would also recommend seeing how these options display on webhost in the tool tips, it can be a little weird sometimes.

Reply:
![image](https://github.com/user-attachments/assets/dcc2d2a5-1728-4405-86ae-f7e9c24d6de5)

Reply:
Yeah there's that little indentation there at "to gain access to HAG1" that looks a little weird